### PR TITLE
FW Update modules for HPE iLO -  ilo_redfish_command.py

### PR DESCRIPTION
**Pull request by rajeevkallur**
##### SUMMARY
This is an enhancement for HPE iLO FW Update operations.  This is a new source file.

##### ISSUE TYPE
FW Update.

##### COMPONENT NAME
ilo_redfish_command

##### ADDITIONAL INFORMATION
Unit testing done.

### CODE REVIEW

**Comment by mraineri** [comment]
What's the difference between using this instead of the existing "SimpleUpdate" command? Does this invoke OEM actions? Could you please provide the flow of how this interacts with a Redfish service?

**Comment by rajeevkallur** [comment]
> What's the difference between using this instead of the existing "SimpleUpdate" command? Does this invoke OEM actions? Could you please provide the flow of how this interacts with a Redfish service?

This HPE OEM specific method of FW update.  It does not use simple update redfish URL

**Comment by mraineri** [comment]
> > What's the difference between using this instead of the existing "SimpleUpdate" command? Does this invoke OEM actions? Could you please provide the flow of how this interacts with a Redfish service?
> 
> This HPE OEM specific method of FW update. It does not use simple update redfish URL

Can we change the name of the command to include HPE-specific terminology? I would like the user to avoid confusing this with standard functionality.

File: plugins/modules/ilo_redfish_command.py [hunk 1]
```diff
@@ -0,0 +1,239 @@
...
+# Copyright (c) 2021-2022 Hewlett Packard Enterprise, Inc. All rights reserved.
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = '''
+---
+module: ilo_redfish_command
+short_description: Sets or updates configuration attributes on HPE iLO with Redfish OEM extensions
```
Comment (felixfontein) [inline plugins/modules/ilo_redfish_command.py:12]:
```suggestion
short_description: Sets or updates configuration attributes on HPE iLO with Redfish OEM extensions
version_added: 6.2.0
```

**Review by felixfontein — COMMENTED** [review]
Please note that all new modules must come with tests (unit and/or integration tests).

**Comment by rajeevkallur** [comment]
Closing this as we wont be merging this with community.general due to the nature of this feature only used by hpe. It will be part of hpe.ilo collection which will be published soon.

**Pull request closed without merging**